src/attacks/attack_execute_malware.py

```diff
@@ -51,7 +51,9 @@ def _execute_malware_command(self):
         client_nr = self.options.rhost[-2:].lstrip("0")
         client_name = "client" + client_nr
 
-        cmd = (f"schtasks /create /sc once /st 23:59 /tn {random_tag} "
+        cmd = (f"if exist {file} "
+               f"(schtasks /create /sc once /st 23:59 /tn {random_tag} "
                f"/tr \"{file}\" /ru BREACH\\{client_name} && "
-               f"schtasks /run /tn {random_tag}")
+               f"schtasks /run /tn {random_tag}) "
+               f"else (echo Unable to locate {file})")
```

Comment:
Did you test if this works as intended when the file is not found?

Reply:
The first part of the command `f"if exist {file} "` checks for the existence of the file before creating the task and running it. Missing file will always cause the attack to fail

Reply:
OK, that's what it is intended to do. But did you test it for the case that the file does not exist? I'd like to avoid untested code.

Reply:
Yep, also misunderstood what you meant here. I explicitly tested all cases, works as intended for each one